### ISSUE TITLE
Accessibility - WCAG2.1 - list without content

### ISSUE THREAD

**Issue by perriard** [snapappointments/bootstrap-select#2571]
Hi

When running WCAG 2.1 level A accessibility tests, the code fails the 1.3.1 success criterion - using ol, ul and dl for lists or groups of links. (See https://www.w3.org/WAI/WCAG21/Techniques/html/H48 )

It fails because the inner dropdown element <ul class="dropdown-menu inner show" role="presentation"></ul> does not have any children <li> - they are not populated until the selectpicker has been clicked.

You can see a test of the examples page http://qualweb.di.fc.ul.pt/evaluator/https%253A%252F%252Fdeveloper.snapappointments.com%252Fbootstrap-select%252Fexamples%252F 
- scroll down to the "failed" result with the title "USING OL, UL AND DL FOR LISTS OR GROUPS OF LINKS"

Is this something you would consider fixing? :-)
